cf/data/data.py

```diff
@@ -10153,7 +10157,8 @@ def floor(self, inplace=False, i=False):
         [-2. -2. -2. -1.  0.  1.  1.  1.  1.]
 
         """
-        return self.func(np.floor, out=True, inplace=inplace)
+        # TODODASK: do we need 'out' specification to 'func'? Was out=True
```

Comment:
In the light of the rewritten `Data.func`, I don't think we need `out` in the `func` call anymore.

Reply:
Thanks for the review, David. With quite a bit to note in the opening comment I forgot to mention 'out' so thanks for picking up on it!

I worked out that it was surely there for the optional kwarg to pass through to NumPy `ufunc`-like functions to specifiy the 'alternate array object(s) in which to put the result' but wasn't entirely sure why it was used in cases such as this. I suspected it could go, so glad to hear that is right. I'll get rid of those with a deprecation warning on the kwarg in a new commit.